mino/minogrpc/example_test.go

```diff
@@ -11,6 +14,110 @@ import (
 	"go.dedis.ch/dela/serde"
 )
 
+func TestRpcStreamTree(t *testing.T) {
```

Comment:
If this is a test and not an example then it will need to be moved to a _test.go file. Could be a new `scenario_test.go` file, or maybe in `mod_test.go`.